src/content/docs/en/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,529 @@
+
+#### What should I do?
+If you want to keep your project's current behavior, set `build.inlineStylesheets` to the previous default, `"never"`:
+
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+	 build: {
+    inlineStylesheets: "never"
+  }
+})
+```
+
+### Changed default: image service
+
+In Astro v2.x, Squoosh was the [default image processing service](/en/guides/images/#default-image-service).
+
+Astro v3.0 now includes Sharp as the default image processing service, and instead provides a configuration option to use Squoosh.
+
+#### What should I do?
+
+If you had previously installed Sharp, uninstall the dependency using your package manager, or by removing manually from your `package.json`. It is now included in Astro.
+
+If you would prefer to continue to use Squoosh to transform your images, update your config with the following: 
+
+```ts title="astro.config.mjs" ins={4-6}
+import { defineConfig, squooshImageService } from "astro/config";
+
+export default defineConfig({
+  image: {
+    service: squooshImageService(),
+  }
+})
+```
+
+### Changed: HTTP request methods case
+
+In Astro v2.x, [HTTP request methods](/en/core-concepts/endpoints/#http-methods) were written using lowercase function names: `get`, `post`, `put`, `all`, and `del`.
+
+Astro v3.0 uses uppercase function names, including `DELETE` instead of `del`.
+
+#### What should I do?
+
+Rename all functions to their uppercase equivalent:
+
+- `get` to `GET`
+- `post` to `POST`
+- `put` to `PUT`
+- `all` to `ALL`
+- `del` to `DELETE`
+
+```js title="endpoint.ts" del={1} ins={2}
+export function get() {
+export function GET() {
+    return new Response(JSON.stringify({ "title": "Bob's blog" }));
+}
+```
+
+### Changed: Multiple JSX framework configuration
+
+In Astro v2.x, you could use [multiple JSX framework integrations](/en/guides/integrations-guide/#official-integrations) (React, Solid, Preact) in the same project without needing to identify which files belonged to which framework.
+
+Astro v3.0 now requires you to specify which framework to use for your files with new `include` and `exclude` integration config options when you have multiple JSX framework integrations installed. This allows Astro to better support single-framework usage, as well as advanced features like React Fast Refresh.
+
+#### What should I do?
+
+If you are using multiple JSX frameworks in the same project, set `include` (and optionally `exclude`) to an array of files and/or folders. Wildcards may be used to include multiple file paths.
+
+We recommend placing common framework components in the same folder (e.g. `/components/react/` and `/components/solid/`) to make specifying your includes easier, but this is not required:
+
+```js
```

Comment:
As with other examples of the page, it would be better to highlight the added code for consistency.

```suggestion
```js ins={11-21}
```

Reply:
Oooh, good call @morinokami! Since the integrations themselves will already be there, I think maybe JUST highlighting the `include` lines make sense? If you update, I'll commit!

Reply:
And, OF COURSE, @morinokami you are absolutely allowed to review this! You are one of our most eagle-eyed contributors! It's not complete until you say so! :purple_heart:

Reply:
Updated my suggestion!!

````suggestion
```js ins={13,16,19}
````